content/6_Advanced/Critical.mdx

```diff
@@ -10,17 +10,318 @@ frequency: 1
 
 ## Paths
 
-CSES Critical Cities https://cses.fi/problemset/task/1703/
+<FocusProblem problem="critical" />
 
-Dominator Tree
+<Resources>
+	<Resource
+		source="Wiki"
+		title="Dominator"
+		url="https://en.wikipedia.org/wiki/Dominator_(graph_theory)"
+		starred
+	>
+  	Wiki Definition
+ 	</Resource>
+	<Resource
+		source="Blog"
+		title="Dominator Tree"
+		url="https://tanujkhattar.wordpress.com/2016/01/11/dominator-tree-of-a-directed-graph/"
+	>
+    Well-covered article
+    </Resource>
+	<Resource
+		source="Blog"
+		title="Visualizing Dominators"
+		url="https://sbaziotis.com/compilers/visualizing-dominators.html"
+		starred
+	>
+    Well-covered article
+    </Resource>
+</Resources>
 
-https://tanujkhattar.wordpress.com/2016/01/11/dominator-tree-of-a-directed-graph/
+## Solution
 
-_ https://codeforces.com/gym/100513/problem/L _
-https://open.kattis.com/problems/skiresort
+## Naive approach
+
+Critical nodes in a DAG are commonly known as **dominators**. Let's define $Dom(u)$ as the set of nodes that dominate node $u$.
+
+The dominator of the starting node $1$ is itself. The set of dominators for any other node $u$ is the intersection of the set of dominators for all ancestors
+$p$ of node $u$.
+
+$$
+Dom(u)=
```

Comment:
texttt{}? i think standard function formatting is to have all lowercase letters right (optional)

Reply:
Idk, \texttt uses monospace and it look kinda weird